modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/util/JavaSourceUtil.java

```diff
@@ -37,6 +41,40 @@
  */
 public class JavaSourceUtil extends SourceUtil {
 
+	public static String addImport(
+			String content, String fileName, String... newImportNames)
+		throws IOException, ParseException {
+
+		JavaClass javaClass = JavaClassParser.parseJavaClass(fileName, content);
+
+		List<String> neededImports = new ArrayList<>();
+
+		List<String> importNames = javaClass.getImportNames();
```

Comment:
@nicolas-sss, can you please declare the variables according to their usage?

Reply:
Yep